app/default/data.py

```diff
@@ -1,42 +1,35 @@
-import json
-import os
-
 import requests
 from config import Config
+from flask import current_app
 
 
 def get_data(endpoint: str):
-    if endpoint.startswith("https://"):
-        data = get_remote_data(endpoint)
+    """
+        Queries the api endpoint provided and returns a
+        data response in json format.
+
+    Args:
+        endpoint (str): an API get data address
+
+    Returns:
+        data (json): data response in json format
+    """
+    if endpoint.startswith("https://") or Config.ALLOW_HTTP_API_CALLS:
```

Comment:
Could we merge https://github.com/communitiesuk/funding-service-design-frontend/pull/53 and use the `USE_LOCAL_DATA` flag instead?

Reply:
yes absolutely 👍